docker/.env

```diff
@@ -381,48 +381,6 @@ SMQ_WS_ADAPTER_HTTP_SERVER_KEY=
 SMQ_WS_ADAPTER_INSTANCE_ID=
 
 ## Addons Services
-### Bootstrap
```

Comment:
Remove L179 `SMQ_BOOTSTRAP_URL=http://bootstrap:9013` from the UI section (we will remove the whole UI or replace it soon).